### PR TITLE
Login (MT-62)

### PR DESCRIPTION
**Pull request by oliabn**
A separate Login App that returns a token in cookies and a response with a username and id

### CODE REVIEW

File: backend/login/views.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+        if user is None:
+            return Response({"Invalid": "Invalid username or password"}, status=status.HTTP_404_NOT_FOUND)
+
+        data = get_tokens(user)
+        # set cookie for response
+        response = Response()
+        response.set_cookie(
+            key=settings.SIMPLE_JWT["AUTH_COOKIE"],
+            value=data["access"],
```
Comment (approximator) [inline backend/login/views.py:48]: Probably we need to send also `refresh` token

File: doc/Accounts.md
Comment (approximator) [inline doc/Accounts.md:1]: Probably, most of this information is already documented in swagger.
Maybe it is better not to duplicate it?

**Review by approximator — CHANGES_REQUESTED** [review]

File: doc/Accounts.md
Comment (oliabn) [reply doc/Accounts.md:1]: Swagger has no information about answers that are not code 200

File: backend/login/views.py (hunk 1 above)
Comment (oliabn) [reply backend/login/views.py:48]: Frontends said that they don't need a refresh token, they said that they will login all times when the access token will be expired. 
I asked them about it several times. They always said they don't need a refresh token

File: backend/login/admin.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
```
[the remark this answers is not in the document]
Comment (oliabn) [reply backend/login/admin.py]: I deleted empty file

File: backend/login/tests.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
```
[the remark this answers is not in the document]
Comment (oliabn) [reply backend/login/tests.py]: I deleted empty file

File: backend/config/urls.py [hunk 4]
```diff
@@ -27,12 +27,14 @@
     SpectacularSwaggerView,
 )
 
-from . import settings
+# from . import settings
```
[the remark this answers is not in the document]
Comment (oliabn) [reply backend/config/urls.py]: Done

File: backend/config/settings.py [hunk 5]
```diff
@@ -151,7 +154,8 @@
     ],
     "DEFAULT_AUTHENTICATION_CLASSES": [
         # JWT authentication
-        "rest_framework_simplejwt.authentication.JWTAuthentication",
+        # "rest_framework_simplejwt.authentication.JWTAuthentication",
```
[the remark this answers is not in the document]
Comment (oliabn) [reply backend/config/settings.py]: Done

File: backend/login/models.py [hunk 6]
```diff
@@ -0,0 +1,2 @@
+
```
[the remark this answers is not in the document]
Comment (oliabn) [reply backend/login/models.py]: I deleted the empty file

File: backend/login/admin.py
[the remark this answers is not in the document]
Comment (oliabn) [reply backend/login/admin.py:1]: I deleted the empty file

File: backend/login/tests.py
[the remark this answers is not in the document]
Comment (oliabn) [reply backend/login/tests.py:1]: I deleted the empty file

**Review by approximator — APPROVED** [review]

**Pull request merged**